src/app/routes/home/getInitialData/index.js

```diff
@@ -14,13 +14,28 @@ const transformJson = pipe(
   filterGroupsWithoutStraplines,
 );
 
-export default async path => {
+export default async (path, service) => {
+  const ssrDataSources = [
+    `${process.env.SIMORGH_BASE_URL}/${service}/bbc_${service}_radio/schedule.json`, // whack this in config and use in both server and client
+    // could also add most read in here
+    // and any more data you need
+  ];
+
   const { json, ...rest } = await fetchPageData(path);
+  const ssrDataPayloads = await Promise.all(
```

Comment:
We could fetch the main page data in parallel too, there's no reason to wait for that promise to resolve here (may be different for CPS assets when you don't know the page type upfront)